Source/g_game.c

```diff
@@ -2476,6 +2492,16 @@ void G_DeferedInitNew(skill_t skill, int episode, int map)
   d_episode = episode;
   d_map = map;
   gameaction = ga_newgame;
+
+  if (demorecording)
+  {
+    char *tmp;
+    G_CheckDemoStatus();
+    tmp = M_StringDuplicate(demoname);
+    G_RecordDemo(tmp);
```

Comment:
Could we change it so that demos are not overwritten when restarting the recording?

Reply:
Add postfix like `demo0001`? I've seen something similar in Crispy's code.

Reply:
Yes, adding an ever increasing number, so we never end up with two demos of the same name overriding each other.